bolt/lib/Rewrite/DWARFRewriter.cpp

```diff
@@ -366,6 +366,12 @@ static cl::opt<bool> AlwaysConvertToRanges(
     cl::desc("This option is for testing purposes only. It forces BOLT to "
              "convert low_pc/high_pc to ranges always."),
     cl::ReallyHidden, cl::init(false), cl::cat(BoltCategory));
+
+cl::opt<std::string> CompDirOverride(
+    "comp-dir-override",
+    cl::desc("Overrides DW_AT_comp_dir which is used with DW_AT_dwo_name to "
+             "construct a path to *.dwo files."),
```

Comment:
To make this options description more user-friendly, would something like "Specify alternative location to search for *.dwo files" be accurate enought?

Reply:
But it's not full location. It's just the root path to which dwo_name is appended to.

Reply:
"Alternative base location" then?